docs/topics/Git&Github.md

```
@@ -0,0 +1,291 @@
+
+### VCS?
+**Version Control System**
+
+수정한 파일을 매번 저장한다면 메모리에 차지하는 용량이 점점 커진다.
+
+- 변경사항만 저장할 수 있다면 좋을텐데?
+
+  *-> 버전 관리를 하자 !*
+
+>git 가이드북
+>https://git-scm.com/book/ko/v2
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)